pkg/agent/openflow/pipeline.go

```diff
@@ -177,10 +177,16 @@ var (
 	L2ForwardingOutTable = newTable("Output", stageOutput, pipelineIP)
 
 	// Tables of pipelineMulticast are declared below. Do don't declare any tables of other pipelines here!
-
+	// Tables in stageEgressSecurity:
+	// Since IGMP Egress rules only support IGMP report which is handled by packetIn, it is not necessary to add
+	// MulticastIGMPEgressMetricTable here.
+	MulticastEgressRuleTable   = newTable("MulticastEgressRule", stageEgressSecurity, pipelineMulticast)
+	MulticastEgressMetricTable = newTable("MulticastEgressMetric", stageEgressSecurity, pipelineMulticast)
 	// Tables in stageRouting:
 	MulticastRoutingTable = newTable("MulticastRouting", stageRouting, pipelineMulticast)
-
+	// Tables in stageIngressSecurity
+	MulticastIGMPIngressTable   = newTable("MulticastIGMPIngress", stageIngressSecurity, pipelineMulticast)
```

Comment:
I wonder if we should just call it MulticastIngressTable. Even if we support multicast ingress rule, they could be in this table instead of a separate table?

Reply:
OK，will rename it.